lib/components/proposal/body/ProposalBody.tsx

```
@@ -20,6 +20,7 @@ interface ProposalBodyProps {
 	proposerInfo: UserProfile | undefined;
 	descriptionLength?: number;
 	origin?: EPostOrigin;
+	withoutReadMore?: Boolean
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)